### PR TITLE
Fix private registry installation procedure

### PR DESCRIPTION
**Pull request by mhoshi-vm**
Fix private registry installation procedure

#45

### CODE REVIEW

File: configuring-sys-image-registry.html.md.erb [hunk 1]
```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
         kbld unpackage -f <(ytt -f config/image-overrides.yml \
                          -f config/values.yml) \
                          -i /tmp/images.tar \
-                         -r registry.SYSTEM-DOMAIN/ORG-NAME/REPOSITORY \
+                         -r PRIVATE_REPOSITORY/PROJECT/IMAGE \
                          --lock-output relocated_images.yml
+        sed -i 's/image: registry.pivotal.io/image: 10.195.35.68/g' relocated_images.yml
```
Comment (julian-hj) [inline configuring-sys-image-registry.html.md.erb:140]: Presumably the IP address here needs to be replaced with some variant of PRIVATE_REPOSITORY?

File: configuring-sys-image-registry.html.md.erb [hunk 2]
```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
...
 
         ```
         kbld unpackage -f <(ytt -f config/image-overrides.yml \
                          -f config/values.yml) \
                          -i /tmp/images.tar \
-                         -r registry.SYSTEM-DOMAIN/ORG-NAME/REPOSITORY \
+                         -r PRIVATE_REPOSITORY/PROJECT/IMAGE \
                          --lock-output relocated_images.yml \
                          --registry-verify-certs=false
+        sed -i 's/image: registry.pivotal.io/image: 10.195.35.68/g' relocated_images.yml
```
Comment (julian-hj) [inline configuring-sys-image-registry.html.md.erb:157]: Again, we need something other than an IP address here

**Review by julian-hj — CHANGES_REQUESTED** [review]
Hi @mhoshi-vm thanks for the suggestions. This PR is a bit difficult to work with however, as we don't have any context about what the problems were that you hit when you tried to follow the original documentation.

In order to help us try to get this documentation right, can you describe the problems you had, and how this revised set of steps makes them better?

Thanks,
-Julian

File: configuring-sys-image-registry.html.md.erb [hunk 3]
```diff
@@ -118,16 +118,13 @@ To upload the TAS for Kubernetes images to the private registry:
 1. Log in to the private registry.  
 
     ```
-    docker login registry.SYSTEM-DOMAIN/ORG-NAME/REPOSITORY \
```
Comment (mhoshi-vm) [inline configuring-sys-image-registry.html.md.erb:121]: This line is wrong, as we should be logging into the docker repository.

File: configuring-sys-image-registry.html.md.erb [hunk 4]
```diff
@@ -118,16 +118,13 @@ To upload the TAS for Kubernetes images to the private registry:
...
+    docker login PRIVATE_REPOSITORY \
                        -u USERNAME \
                        -p PASSWORD
     ```
 
     Where:
-    * Where `SYSTEM-DOMAIN` is the system domain, such as `example.com`, that you previously selected for your
```
Comment (mhoshi-vm) [inline configuring-sys-image-registry.html.md.erb:127]: Updated this to only PRIVATE_REPOSITORY

File: configuring-sys-image-registry.html.md.erb [hunk 5]
```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
         kbld unpackage -f <(ytt -f config/image-overrides.yml \
                          -f config/values.yml) \
                          -i /tmp/images.tar \
-                         -r registry.SYSTEM-DOMAIN/ORG-NAME/REPOSITORY \
```
Comment (mhoshi-vm) [inline configuring-sys-image-registry.html.md.erb:141]: There is no registry running on registry.SYSTEM-DOMAIN so this has been updated

File: configuring-sys-image-registry.html.md.erb [hunk 6]
```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
...
+                         -r PRIVATE_REPOSITORY/PROJECT/IMAGE \
                          --lock-output relocated_images.yml
+        sed -i 's/image: registry.pivotal.io/image: PRIVATE_REPOSITORY/g' relocated_images.yml
         ```
 
         Where:
-        * Where `SYSTEM-DOMAIN` is the system domain, such as `example.com`, that you previously selected for your 
-        TAS for Kubernetes installation.
-        * `ORG-NAME` is your organization name, such as a Cloud Foundry Org name.
-        * `REPOSITORY` is the private registry repository to log in to.
```
Comment (mhoshi-vm) [inline configuring-sys-image-registry.html.md.erb:149]: Updated comment

File: configuring-sys-image-registry.html.md.erb [hunk 7]
```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
...
+        * `PROJECT` is the project name in your private registry
+        * `IMAGE` is the image name to reallocate the images.
 <br>
     * If your client does not have the private registry CA certificates:  
 
         ```
         kbld unpackage -f <(ytt -f config/image-overrides.yml \
                          -f config/values.yml) \
                          -i /tmp/images.tar \
-                         -r registry.SYSTEM-DOMAIN/ORG-NAME/REPOSITORY \
```
Comment (mhoshi-vm) [inline configuring-sys-image-registry.html.md.erb:157]: Same as above

File: configuring-sys-image-registry.html.md.erb [hunk 8]
```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
...
+                         -r PRIVATE_REPOSITORY/PROJECT/IMAGE \
                          --lock-output relocated_images.yml \
                          --registry-verify-certs=false
+        sed -i 's/image: registry.pivotal.io/image: PRIVATE_REPOSITORY/g' relocated_images.yml
         ```
 
         Where:
-        * Where `SYSTEM-DOMAIN` is the system domain, such as `example.com`, that you previously selected for your
```
Comment (mhoshi-vm) [inline configuring-sys-image-registry.html.md.erb:163]: Same as above

File: configuring-sys-image-registry.html.md.erb [hunk 9]
```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
...
+        sed -i 's/image: registry.pivotal.io/image: PRIVATE_REPOSITORY/g' relocated_images.yml
         ```
 
         Where:
-        * Where `SYSTEM-DOMAIN` is the system domain, such as `example.com`, that you previously selected for your 
-        TAS for Kubernetes installation.
-        * `ORG-NAME` is your organization name, such as a Cloud Foundry Org name.
-        * `REPOSITORY` is the private registry repository to log in to.  
-
-### <a id='configure-for-private-registry'></a> Customize Your Configuration to Use Your Private System Registry
```
Comment (mhoshi-vm) [inline configuring-sys-image-registry.html.md.erb:168]: This section does not look accurate and did not work for me i will add the reasons

File: configuring-sys-image-registry.html.md.erb [hunk 10]
```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
...
-
-To customize your configuration files for your private registry: 
-
-1. Move the `config/image-overrides.yml` file to the `config-optional` directory.
```
Comment (mhoshi-vm) [inline configuring-sys-image-registry.html.md.erb:172]: During "kpack package" all images are renamed to "registry.pivotal.io". 
The reallocation file created during "kpack unpackage" is based on the assumption that images are all from "registry.pivotal.io". 
Without this image-overrides, we are unable to convert the images via kbld properly as it is needed to covert to registry.pivotal.io

File: configuring-sys-image-registry.html.md.erb [hunk 11]
```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
...
-1. To point the system registry variables to the private registry,
```
Comment (mhoshi-vm) [inline configuring-sys-image-registry.html.md.erb:173]: Changing `configuration-values/values.yml` has no affect to the later procedure as it get's overwritten to what it has at "../configure-values/system-registry.yaml"

File: installing-tas-for-kubernetes.html.md.erb [hunk 12]
```diff
@@ -79,23 +79,16 @@ To install TAS for Kubernetes:
 1. Change directory to the `tanzu-applications-service` directory.
 
 1. To deploy TAS for Kubernetes using the images in the private registry:
-    * If your client has the private registry CA certificates:
```
Comment (mhoshi-vm) [inline installing-tas-for-kubernetes.html.md.erb:82]: The kbld command does not connect to registry, so there is no meaning to add certificate ignore flags

File: installing-tas-for-kubernetes.html.md.erb [hunk 13]
```diff
@@ -79,23 +79,16 @@ To install TAS for Kubernetes:
...
-
-        ```
+    ```
+    kbld -f <(  \
         ytt -f config \
-              -f ../configuration-values | \
-              kbld -f- -f relocated_images.yml > /tmp/final_deployment.yml \
-              --registry-verify-certs=false
-        kapp deploy -a cf -f /tmp/final_deployment.yml
-        ```
```
Comment (mhoshi-vm) [inline installing-tas-for-kubernetes.html.md.erb:98]: The original command does not work as exepcted, and i was only able to find work with the updated version

**Comment by mhoshi-vm** [comment]
hi @julian-hj 
I added as much as context as possible, but as you see , it needs major rewrite.
There is too many that is not accurate in the procedure, at least has not work in my envs so far

**Pull request merged**